src/System.IO.FileSystem/tests/Performance/System.IO.FileSystem.Performance.Tests.csproj

```diff
@@ -26,6 +31,9 @@
     <Compile Include="$(CommonTestPath)\System\IO\PathFeatures.cs">
       <Link>Common\System\IO\PathFeatures.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
```

Comment:
orthogonal but I want PlatformDetection to be in the TestUtilities as well at some point.